test/unit/alignment/aligned_sequence_test_template.hpp

```diff
@@ -15,12 +16,9 @@
 #include <seqan3/core/detail/debug_stream_alphabet.hpp>
 #include <seqan3/core/detail/debug_stream_range.hpp>
 #include <seqan3/io/alignment_file/detail.hpp>
-#include <seqan3/std/iterator>
 
 using seqan3::operator""_dna4;
 
-using namespace seqan3; // necessary right now because assign_unaligned is falsely a hidden friend
```

Comment:
this is not needed anymore?

Reply:
This line was commented by @h-2, but all tests were all running through without this line. ¯\_(ツ)_/¯

Reply:
It's true that thiscan be removed, but it is not possible to write `seqan3::assign_unaligned()` instead of just `assign_unaligned()` which would be the correct way.

Reply:
Time for git bisect :D

Reply:
I've sent the core team my thought on how this should be designed with CPOs...